pages/credits.html

```diff
@@ -136,7 +134,11 @@ <h1 id="title-header">The Regis Altare Birthday Project 2023</h1>
         </button>
     </nav>
 
-      <section id="credits"></section>
+      <section id="credits">
+
+        <h1>Project Credits</h1>
```

Comment:
(nit) you can link the `h1` to `section` with aria-labelledby so that when a screenreader is browsing by section, the section will be read out like "Section: Project Credits" instead of just "Section"

So like
```
      <section aria-labelledby="credits-header" id="credits">

        <h1 id="credits-header">Project Credits</h1>
```

https://accessibility.psu.edu/screenreaders/voiceover/ (see 'Rotor Window' - it's a convenient function that lets users jump to specific headers/ sections, which is why it's important to use those HTML tags. It's something like ctrl + cmd + U on VoiceOver)